Tech Lead/Minutes/20241112.md

```diff
@@ -0,0 +1,27 @@
+# Technical Lead Development Team call 12 November 2024
+
+## Participants
+
+- Attendees: @schlunma @bouweandela @rbeucher @valeriupredoi @charles-turner-1
+- Absentees: @flicj191 @ehogan @sloosvel
+- Chair: @valeriupredoi
+- Minutes taker: @valeriupredoi
+
+## Action Items
+
+| Meeting | Assignees | Item | Status |
+|-|-|-|-|
+|2024-10-01| @ehogan | Open an Issue or Discussion item about geospatial benchmark suite | **Done**: [ESMValTool: Discussion #3764: Would we like to help create a geospatial benchmark suite?](https://github.com/ESMValGroup/ESMValTool/discussions/3764)
+|2024-08-13| all | Review [RTW pull request](https://github.com/ESMValGroup/ESMValTool/pull/3210) | pending |
+
```

Comment:
It looks like a couple of action items are missing, i.e. the `pending` ones from the [previous meeting](https://github.com/ESMValGroup/Community/discussions/192#discussioncomment-11206252). We did discuss those, right?

Reply:
aye, good spot, added them in ea181ae3906839b0e258526918982e207515a4f6 :beer:

Reply:
Thanks! But since we discussed them and decided they were done, should we mark them as such?

Reply:
cheers, bud :beer: